app/src/main/java/org/schabi/newpipe/fragments/list/search/SearchFragment.java

```diff
@@ -837,7 +836,10 @@ public void showError(String message, boolean showRetryButton) {
 
     @Override
     public void handleResult(@NonNull SearchInfo result) {
-        if (!result.getErrors().isEmpty()) {
+        final List<Throwable> exceptions = result.getErrors();
+        if (!exceptions.isEmpty()
+            && !(exceptions.size() == 1
+                && exceptions.get(0) instanceof SearchExtractor.NothingFoundException)){
```

Comment:
I thought it was including additional information.

Reply:
Seems like bad practice if it's just there to speed up the collector process. But this is a problem for another PR, I suppose.

Reply:
> But this is a problem for another PR, I suppose.

I see. Another thing I thought of was that if the search result is empty, but no error is thrown, it could be an error that did not get detected, but if we tell the fronted explicitly that there was nothing found, than at least we know there was no error. Maybe we can implement that without an exception, but by setting a flag.